src/stylesheets/elements/_tags.scss

```diff
@@ -89,29 +89,46 @@
   border: 1px solid #981b1e;
   background-color: #e59393;
 }
+//Border colors
+.sds-tag--border-blue {
+  border-color:  #00a6d2!important;
+  border-radius: 3px;
+}
+.sds-tag--border-red {
+  border-color:  #981b1e!important;
+  border-radius: 3px;
+}
+.sds-tag--border-yellow {
+  border-color:  #fdb81e!important;
+  border-radius: 3px;
+}
```

Comment:
We should use the status names (error, info, alert) instead of generic colors here. Also, let's try to use the includes: @include u-border('primary-vivid') and figure out how to not use !important if possible. 

For radius, let's verify with designers that they intended to make this variant the only rounded one, if so use:
@include u-radius

Reply:
I am using tokens now